docs/user/getting_started.rst

```diff
@@ -25,17 +25,18 @@ An example configuration can be found in the Gluon repository at *docs/site-exam
 Dependencies
 ------------
 To build Gluon, several packages need to be installed on the system. On a
-freshly installed Debian Stretch system the following packages are required:
+freshly installed Debian Bullseye system the following packages are required:
 
 * `git` (to get Gluon and other dependencies)
-* `subversion`
 * `python3`
 * `build-essential`
+* `ecdsautils`
```

Comment:
I'm unsure whether ecdsautils should be considered a dependency. Maybe with a note that this is only needed for `contrib/sign.sh`?

Reply:
I added a note that this is only needed to sign firmware.
I think it makes it easier to understand the autoupdater procedure if this dependency is already listed here.